android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -203,8 +203,12 @@ public void processProperties(KrollDict d)
 		if (d.containsKey(TiC.PROPERTY_BACKGROUND_COLOR)) {
 			// Why transparent?
 			tv.setBackgroundColor(Color.TRANSPARENT);
-			textInputLayout.setBoxBackgroundColor(TiConvert.toColor(d.get(TiC.PROPERTY_BACKGROUND_COLOR),
-				TiApplication.getAppCurrentActivity()));
+			if (d.get(TiC.PROPERTY_BACKGROUND_COLOR).equals("transparent")) {
```

Comment:
- This would probably crash if the `backgroundColor` is passed `null`. A safe condition would be this `"transparent".equals(d.get(TiC.PROPERTY_BACKGROUND_COLOR)))` using the `equals()` on the literal.
- Also, passing the `null` should logically be equal to passing a `transparent` value, right?

Reply:
I've checked it with 12.6.0 and `null` was transparent there too, so you are right. Updated the code and screenshots.